src/ui/ResponsiveComponents/ResponsiveDropdown/ResponsiveDropdown.ts

```diff
@@ -77,13 +83,18 @@ export class ResponsiveDropdown {
     this.dropdownHeader.close();
     this.dropdownContent.hideDropdown();
     this.hidePopupBackground();
+    this.unlockScroll();
     $$(this.dropdownHeader.element).trigger(ResponsiveDropdownEvent.CLOSE);
   }
 
   public disablePopupBackground() {
     this.popupBackgroundIsEnabled = false;
   }
 
+  public enableScrollLocking() {
```

Comment:
We should add unit test(s) that capture how toggling `lockScrollIsEnabled` changes the scroll behaviour.

Reply:
Good point, I'll keep that in mind when adding UTs.